### ISSUE TITLE
RPM feedback-  A question

### ISSUE THREAD

**Issue by ne9n** [CircuitFlyer/Climb_and_Dive#1]
First of all nicely done! I have been working for over 30 years in industrial motor control, power tool control, and now lawn equipment.    I have my own Arduino-based pitch controller that I have developed.  I never really got it past prototype.   Mine is all feed-forward and uses the MPU 6050 and SPI and no active feedback.

Do I understand it correctly that the RPM feedback is filtered so you get a single pulse per electrical cycle? 
   
Do you work about high S counts and motor back EMF damaging the comparator?

**Comment by CircuitFlyer** [comment]
Thank you.  I'm no embedded systems expert, just a hack that cobbled together the necessary parts to make it work reasonably well.

Yes, the RPM signal conditioning circuit outputs a single pulse per electrical cycle, electrical RPM per se.

The circuit contains a resistor divider to initially lower the motor winding voltage.  The RC filtering further attenuates the signal voltage feed to the comparator.   It should be able to handle 2S up to 8S lipo batteries.  That seems to be the practical limit of the batteries currently used in control line.

**Comment by ne9n** [comment]
I might suggest some clamping diodes or tvs diodes to clamp the max
voltage.  I can sketch it up if you like.

**Comment by CircuitFlyer** [comment]
Of course, any suggestions to help better protect the circuitry would be welcome.
